packages/vm/src/evm/interpreter.ts

```diff
@@ -261,27 +259,27 @@ export default class Interpreter {
   }
 
   // Returns all valid jump and jumpsub destinations.
-  _getValidJumpDests(code: Buffer): JumpDests {
-    const jumps = []
-    const jumpSubs = []
+  _getValidJumpDests(code: Buffer) {
+    const jumps = new Uint8Array(code.length).fill(0)
```

Comment:
This creates a byte array while Geth uses a bitmap (so 1/8 length of the array is used there). I'm not sure if this is faster or practical in JS but worth mentinoing here.

Reply:
I don't think the bitmaps thing is relevant to us.  Geth is flipping bits for every push (or flipping a bunch at once) where as we don't do any writes at all for `PUSH` but just skip over however many `PUSH`s there are.

Reply:
Agreed